.github/workflows/npm-publish.yml

```diff
@@ -12,17 +10,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout repository
```

Comment:
Included in the suggestion on my other comment (couldn't suggest the whole thing because it spanned deleted lines)
```suggestion
```